src/index.html

```diff
@@ -17,6 +21,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="moyologo.png"
+          alt="Moyo logo"
+          class="img"
+        />
+      </a>
+      <nav>
+        <ul class="navlist">
+          <li class="style">
+            <a
+              href="#"
+              class="nav-link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="style">
+            <a
+              href="#"
+              class="nav-link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="style">
+            <a
+              href="#"
+              class="nav-link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li class="style">
+            <a
+              href="#"
+              class="nav-link for-test:hover"
```

Comment:
The class name 'for-test:hover' is not a valid class name because it includes a pseudo-class ':hover'. Class names should not contain colons or other special characters. Consider renaming this class to something more appropriate, like 'hover-test'.